flow/designs/gf12/jpeg/rules-base.json

```diff
@@ -62,13 +62,13 @@
         },
         {
             "field": "finish__timing__setup__ws",
-            "value": 0.0,
+            "value": -25.18,
```

Comment:
I thought this was just going to be a small CTS update - what happened here?

Reply:
The file was manually updated by @tspyrou last time. See [here](https://github.com/The-OpenROAD-Project/OpenROAD-flow-scripts/tree/master/flow/designs/gf12/jpeg). Thus, now when I updated using `make update_ok` other rules got changed as well.